### ISSUE TITLE
log fills up when mpd isn't running

### ISSUE THREAD

**Issue by goodevilgenius** [kori/wylt#2]
When `mpd` is not running, but libra is, the libra log fills up with lines like this:

```
2019/02/05 06:56:41 write tcp 127.0.0.1:36040->127.0.0.1:6600: write: broken pipe
```

I left this overnight, and the log had become over 100GB in size.

**Comment by kori** [comment]
I was wrong, I started editing the code  and it seems this is what produces the error:

https://github.com/kori/libra/blob/e06a06e515ddcfac65a76498e70a7c38229085a8/players/mpd/mpd.go#L72-L76

A fix should be pushed today.

**Comment by quite** [comment]
I'm getting nothing in the log, but libra is using a lot of cpu when mpd is not playing (and causes mpd to also use cpu at the same time). Wonder if this is related...

**Comment by kori** [comment]
@quite I think it might be related, but i'm not sure how to fix this. I think I might just remove the code block referenced in https://github.com/kori/libra/issues/2#issuecomment-461082198, but I'm not sure.

I'm open to suggestions.

**Comment by kori** [comment]
Solved in edde782
